templates/android/library/build.gradle.twig

```diff
@@ -59,10 +59,7 @@ dependencies {
     api("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.7.1")
     api("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.7.1")
 
-    api(platform("com.squareup.okhttp3:okhttp-bom:4.12.0"))
-    api("com.squareup.okhttp3:okhttp")
-    implementation("com.squareup.okhttp3:okhttp-urlconnection")
-    implementation("com.squareup.okhttp3:logging-interceptor")
```

Comment:
Why did we add these previously? removing these will not cause any issues?

Reply:
> api("com.squareup.okhttp3:okhttp")

We're still using this, just in `implementation` scope instead of `api`. Means that consumers of the SDK won't have direct access to this libraries code. I think I had added this for developing realtime.

>  implementation("com.squareup.okhttp3:okhttp-urlconnection")

Not sure why we had this, we don't need it. This let's you build a Java native `HttpUrlConnection` using okhttp, which we never do.

> implementation("com.squareup.okhttp3:logging-interceptor")

This is a logger for HTTP requests, also added for development testing.